torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -37,14 +37,14 @@ namespace channel {
 class Context;
 } // namespace channel
 
-using DeviceMap = std::unordered_map<c10::DeviceIndex, c10::DeviceIndex>;
-
```

Comment:
Not sure if there was a reason to define this in the `tensorpipe` namespace, which was otherwise used just for forward-declaring TensorPipe's public API. I think it makes more sense for this to be in the `torch` namespace.